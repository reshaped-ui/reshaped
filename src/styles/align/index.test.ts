import getAlignStyles from "./index";

describe("Styles/Align", () => {
	test("handles value", () => {
		expect(getAlignStyles("start")).toMatchSnapshot();
	});

	test("handles undefined value", () => {
		expect(getAlignStyles()).toMatchSnapshot();
	});

	test("handles responsive value", async () => {
		expect(getAlignStyles({ s: "start", m: "center", l: "end" })).toMatchSnapshot();
	});
});
