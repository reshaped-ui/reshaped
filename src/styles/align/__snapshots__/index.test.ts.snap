// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Styles/Align > handles responsive value 1`] = `
{
  "variables": {
    "--rs-align-l": "end",
    "--rs-align-m": "center",
    "--rs-align-s": "start",
  },
}
`;

exports[`Styles/Align > handles undefined value 1`] = `null`;

exports[`Styles/Align > handles value 1`] = `
{
  "variables": {
    "--rs-align-s": "start",
  },
}
`;
